spring-ai-core/src/main/java/org/springframework/ai/reader/TextReader.java

```diff
@@ -88,12 +88,11 @@ public List<Document> get() {
 			this.customMetadata.put(SOURCE_METADATA, this.resource.getFilename());
 
 			return List.of(new Document(document, this.customMetadata));
-			// return textSplitter.apply(Collections.singletonList(new Document(document,
-			// this.customMetadata)));
```

Comment:
The comment was left intentionally for smooth recovery, so it seems inappropriate to remove it.